mentoring/programs/contributor-workshop/resources/guides/videoguide.md

```diff
@@ -0,0 +1,397 @@
+* ``YYYY-MM-DD`` being date, is the date video was recorded
+* ``<Segement Name>`` being the name of the markdown file associated with that segment. see [CONTRIBUTING.md](https://github.com/kubernetes/contributor-site/tree/workshop/content/en/workshop/CONTRIBUTING.md#list-of-segments)
+* ``<gh-handle>`` being the github handle of the person who recorded the video, if there are multiple presenters just add another dash and the handle. ``<gh-handle>-<gh-handle>``
+
+For example
+
+``2021-05-10-why-contribute-alisondy.mp4`` 
+``2021-05-10-why-contribute-alisondy-mrbobbytables.mp4`` 
+
+### Update Google Drive
+
+#### Uploading your video
+
+You will upload your recording to the folder associated with your segment. More details in [What folder to upload to](#what-folder-to-upload-to)
+
+##### What folder to upload to
+There should be a folder for your segment. If one doesn't exist, create a new one with the segment name.
+
+segment name being the name of the markdown file associated with that segment. see [CONTRIBUTING.md](https://github.com/kubernetes/contributor-site/tree/workshop/content/en/workshop/CONTRIBUTING.md#list-of-segments)
+
+For example
+
+``why-contribute``
+
+### Updating Youtube
+
+Once you have uploaded the video to drive, upload it to youtube, and update the previous recording on youtube ( if any )
+
+#### Upload the new video
+
+In youtube, upload the same video and title it as follows
+
+```txt
+Contributor Workshop : <Segment Name> with <Presenter Name> | <Month> <Year>
+```
+
+For example
+
+```txt
+Contributor Workshop : Why Contribute? with @alisondy | May 2021
+```
+
+##### Adding the thumbnail image
+
+```markdown
+TODO : Instructions on how to generate a thumbnail image, similar to https://gist.github.com/alisondy/e4ff3b483a32330ade0c880b4a4a10b1
+```
+
+Set the thumbnail to the image generated in the previous step
+
+##### Setting the description
+
+Use the following template for setting the video description
+
+```txt
+This is the video guide for the Contributor Workshop segment <Segment Name>
+
+Link to this segments page:
+https://kubernetes.dev/docs/workshop/<page-name>
+
+Link to the workshop:
+https://kubernetes.dev/docs/workshop/
+```
+
+#### Update the previous video for the segment if it exists
+
+If you're updating an already existing video, follow the next steps to update the following on the previous video
+- title
+- thumbnail
+- description
+- comments section
+
+##### Update the title
+Update the title on the previous video to match the following format
+
+```txt
+Contributor Workshop : <Segment Name> | <Month> <Year> | 🚨🚨 DEPRECATED 🚨🚨
+```
+
+For example
+
+```txt
+Contributor Workshop : Why Contribute? with @alisondy | May 2021 | 🚨🚨 DEPRECATED 🚨🚨
+```
+
+##### Update the thumbnail image
+
+Update the Thumbnail image on the previous video to
+
+insert-deprecation-thumbnail-here.jpeg
+```txt
+
+
+THIS VIDEO IS DEPRECATED!
+See the comments & description for details on where to find the new video guide!
```

Comment:
There may not be a new video, see the comments and description for more information